example_experiment_config.yaml

```diff
@@ -30,36 +30,15 @@ temporal_config:
 
 
 # COHORT CONFIG
-# To define what entities make it into different matrices, there are three options (only choose one).
+# Cohorts are configured by passing a query with placeholders for the 'as_of_date'.
 #
-# 1. Pass an entities table. All distinct entities present in this table (the 'entity_id' column) will be included in all matrices. Other columns will be ignored
+# The 'query' key should have a query, parameterized with an '{as_of_date}', to select the entity_ids that should be included for a given date. The {as_of_date} will be replaced with each as_of_date that the experiment needs. The returned 'entity_id' must be an integer.
```

Comment:
I'm not sure how widely Triage has standardized on Python string template interpolation in its configurable queries -- and it's not unreasonable -- but standardizing on SQL-formatted and -escaped interpolation would probably make more sense, particularly for the configurable interface:

    :as_of_date

_e.g._:

    engine.execute('select * from atable where as_of_date=:as_of_date', as_of_date=...)

Reply:
This is worth considering, though I would definitely punt on this decision for a refactoring PR as it would change the interface and break old configs.